presto-tpch/src/main/java/com/facebook/presto/tpch/TpchConnectorFactory.java

```diff
@@ -32,16 +32,25 @@
 public class TpchConnectorFactory
         implements ConnectorFactory
 {
+    public static final boolean DEFAULT_IS_PARTITIONING_ENABLED = false;
```

Comment:
How could it break something if it is implemented properly? I just don't like added complexity if it's not necessary.